helm/charts/hpe-csi-driver/values.yaml

```diff
@@ -9,7 +9,7 @@ imagePullPolicy: 'Always'
 backendType: nimble
 
 secret:
-  # parameters for nimble
+  # parameters for specified backendType (nimble, hpe3parprimera)
```

Comment:
can you change this to 3parprimera too?

Reply:
Modified to primera3par, since as per the kubernetes limitation kind name can not start with number